### ISSUE TITLE
vue-i18n@11.0.1 and @intlify/unplugin-vue-i18n@6.0.2 - update package.json peerDependencies

### ISSUE THREAD

**Issue by volarname** [intlify/bundle-tools#431]
### Reporting a bug?

vue-i18n is listed by your project with version 11.0.1 (pdfe07), which doesn't satisfy what @intlify/vue-i18n-extensions (via @intlify/unplugin-vue-i18n) and other dependencies request 

### Expected behavior

it should work without error - package.json requirements needs to be updated

### Reproduction

just upgrade to vue-i18n@11.0.1 and use latest @intlify/unplugin-vue-i18n@6.0.2

### Issue Package

unplugin-vue-i18n

### System Info

```shell
System:
    OS: Linux 5.15 Ubuntu 22.04.3 LTS 22.04.3 LTS (Jammy Jellyfish)
    CPU: (4) x64 13th Gen Intel(R) Core(TM) i9-13900HX
    Memory: 7.89 GB / 15.62 GB
    Container: Yes
    Shell: 5.1.16 - /bin/bash
  Binaries:
    Node: 22.10.0 - ~/.nvm/versions/node/v22.10.0/bin/node
    Yarn: 4.5.3 - ~/.nvm/versions/node/v22.10.0/bin/yarn
    npm: 10.9.0 - ~/.nvm/versions/node/v22.10.0/bin/npm
  npmPackages:
    vite: 6.0.6 => 6.0.6
    vue: 3.5.13 => 3.5.13
    vue-i18n: 11.0.1 => 11.0.1
```

### Additional context

why when new version is released also other bundle tools are not uptaded?

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/intlify/bundle-tools/blob/main/.github/contributing.md).
- [X] Read the README
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/intlify/bundle-tools/discussions).

**Comment by Mertico** [comment]
+1
```
packages/frontend
└─┬ @intlify/unplugin-vue-i18n 6.0.2
  └─┬ @intlify/vue-i18n-extensions 7.0.0
    ├── ✕ unmet peer @intlify/shared@"^9.0.0 || ^10.0.0": found 11.0.1
    └── ✕ unmet peer vue-i18n@"^9.0.0 || ^10.0.0": found 11.0.1
```

**Comment by kazupon** [comment]
related issue

https://github.com/intlify/vue-i18n-extensions/issues/300
We are working on https://github.com/intlify/vue-i18n-extensions/pull/301

**Comment by kazupon** [comment]
This issue has been resolved by #432 
Please try to install latest version.
If you have sitll issue, please put on comments.
Thanks!
